pypeman/remoteadmin.py

```diff
@@ -140,19 +140,34 @@ def get_channel(self, name):
             'status': channels.BaseChannel.status_id_to_str(chan.status)
         }
 
-    async def list_msg(self, channel, start=0, count=10, order_by='timestamp'):
+    async def list_msg(self, channel, start=0, count=10, order_by='timestamp', mk_b64pickle=False):
         """
         List first `count` messages from message store of specified channel.
 
         :params channel: The channel name.
+        :param mk_b64pickle: if True (yield payload / ctx fields as b64
```

Comment:
What if we create an `payload_serializer` param that can be what we want ? The default one is a b64pickle, but we can specify the "jsonable" one ?

Reply:
That's an option.

Another one would be to just pass a 'name' of a serializer and allow to register other ones if desired.

Reply:
what to do?

Reply:
Agree with your proposal.